inbm/dispatcher-agent/dispatcher/aota/aota_command.py

```diff
@@ -283,7 +283,7 @@ def load(self) -> None:
             f'Package: {self._uri} Fetch Result: {result}')
 
         if result.status != CODE_OK:
-            raise AotaError(result.message)
+            raise AotaError(f"Unable to download docker image. Status = {result.status}, message = {result.message}.")
```

Comment:
<h3>$\color{#FFD700}Coding Style: CodeConsistency$</h3>

Consider using a consistent error message format across different methods. The message "Unable to download docker image." and "Unable to download docker-compose container." could be standardized to improve maintainability and readability.